optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/persistence/ConferenceSchedulingImporter.java

```diff
@@ -0,0 +1,496 @@
+        importRoomList();
+        importSpeakerList();
+        importTalkList();
+        importTimeslotList();
+
+        return solution;
+    }
+
+    private String getConferenceName() {
+        logger.info("Sending a request to: " + endpoints.getBaseUrl());
+        JsonObject conferenceObject = readJson(endpoints.getBaseUrl(), JsonReader::readObject);
+        return conferenceObject.getString("label");
+    }
+
+    private void importTalkTypeList() {
+        this.talkTypeNameToTalkTypeMap = new HashMap<>();
+        List<TalkType> talkTypeList = new ArrayList<>();
+        logger.info("Sending a request to: " + endpoints.getBaseUrl() + endpoints.getTalkTypesEndpoint());
+        JsonObject rootObject = readJson(endpoints.getBaseUrl() + endpoints.getTalkTypesEndpoint(), JsonReader::readObject);
+        JsonArray talkTypeArray = rootObject.getJsonArray("proposalTypes");
+        for (int i = 0; i < talkTypeArray.size(); i++) {
+            JsonObject talkTypeObject = talkTypeArray.getJsonObject(i);
+            String talkTypeName = talkTypeObject.getString("label");
+            if (talkTypeNameToTalkTypeMap.keySet().contains(talkTypeName)) {
+                logger.warn("Duplicate talk type in " + endpoints.getBaseUrl() + endpoints.getTalkTypesEndpoint()
+                        + " at index " + i + ".");
+                continue;
+            }
+
+            TalkType talkType = new TalkType((long) i, talkTypeName);
+            talkType.setCompatibleRoomSet(new HashSet<>());
+            talkType.setCompatibleTimeslotSet(new HashSet<>());
+
+            talkTypeList.add(talkType);
+            talkTypeNameToTalkTypeMap.put(talkTypeName, talkType);
+        }
+
+        TalkType breakTalkType = new TalkType((long) talkTypeArray.size(), "break");
+        breakTalkType.setCompatibleRoomSet(new HashSet<>());
+        breakTalkType.setCompatibleTimeslotSet(new HashSet<>());
+        talkTypeList.add(breakTalkType);
+        talkTypeNameToTalkTypeMap.put("break", breakTalkType);
+
+        solution.setTalkTypeList(talkTypeList);
+    }
+
+    private void importTrackIdSet() {
+        this.trackIdSet = new HashSet<>();
+        logger.info("Sending a request to: " + endpoints.getBaseUrl() + endpoints.getTracksEndpoint());
+        JsonObject rootObject = readJson(endpoints.getBaseUrl() + endpoints.getTracksEndpoint(), JsonReader::readObject);
+
+        JsonArray tracksArray = rootObject.getJsonArray("tracks");
+        for (int i = 0; i < tracksArray.size(); i++) {
+            trackIdSet.add(tracksArray.getJsonObject(i).getString("id"));
+        }
+    }
+
+    private void importRoomList() {
+        this.roomIdToRoomMap = new HashMap<>();
+        List<Room> roomList = new ArrayList<>();
+
+        logger.info("Sending a request to: " + endpoints.getBaseUrl() + endpoints.getRoomsEndpoint());
+        JsonObject rootObject = readJson(endpoints.getBaseUrl() + endpoints.getRoomsEndpoint(), JsonReader::readObject);
```

Comment:
remove endpoints.getRoomEndpoint() and simply use

String roomsUrl = endpoints.getBaseUrl() + "/rooms/";